src/BizHawk.Emulation.Cores/Consoles/3DO/LibOpera.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System.Runtime.InteropServices;
+using BizHawk.BizInvoke;
+using BizHawk.Emulation.Cores.Waterbox;
+
+namespace BizHawk.Emulation.Consoles._3DO

```

Comment:
Don't like the underscore in namespace.

Reply:
I couldn't come up with a better solution. Namespaces or variables cannot start with a number

Reply:
ThreeDO

Reply:
whatever you guys decide is better. I personally think _3DO is much closer both alphabetically and visually than ThreeDO.

Reply:
fixed -> ThreeDO